src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ActionPlanSessionController.kt

```diff
@@ -40,7 +40,8 @@ class ActionPlanSessionController(
       updateAppointmentDTO.durationInMinutes,
       user,
       updateAppointmentDTO.appointmentDeliveryType,
-      updateAppointmentDTO.appointmentDeliveryAddress
+      updateAppointmentDTO.appointmentDeliveryAddress,
+      updateAppointmentDTO.deliusOfficeLocationCode,
```

Comment:
i saw some discussion about the best way to refer to this data - is this what we settled on vs nps_office_code or whatever else was discussed?

Reply:
Sorry it's been a while since I've revisted this PR. I've changed back to the usage of npsOfficeCode, I was originally thinking we'd want to make it more generic.